docs/README.md

```diff
@@ -22,7 +22,7 @@ Country data is sorted according to [Figma designs](https://www.figma.com/file/u
 
 ## Place Details
 
-The Place Details component displays a summary of a place according to the address it receives from the wrapper orchestrator, and the description of the structure that receives as prop. There are three variations of PlaceDetails according to the amount of information that needs to be shown in a particular scenario. Below you can see a description of them.
+The Place Details component displays a summary of a place according to the address from the wrapper orchestrator or a custom address passed via props. There are three variations of PlaceDetails according to the amount of information that needs to be shown in a particular scenario. Below you can see a description of them.
```

Comment:
[@praise](https://conventionalcomments.org): good to see docs being updated!

Reply:
thanks :^D

btw, [this bot](https://github.com/vtex-apps/place-components/pull/47#issuecomment-790680631) helps a lot to keep the documentation updated by forcing the PR creator to explicitly define the status of the docs

[See more about it](https://github.com/vtex-apps/docs-bot)

Reply:
Awesome! That's really good to know that we have this solution. But we have some solution for [pending docs](https://github.com/vtex-apps/io-documentation/issues?page=4&q=is%3Aissue+is%3Aopen)?

Reply:
We don't. It only creates the issue and never notifies the codeowners again 😞. Maybe this bot could be improved to send a warning from time to time? Does that make sense to you?